sdk/src/test/java/com/uid2/sdk/data/TestData.kt

```diff
@@ -0,0 +1,15 @@
+package com.uid2.sdk.data
+
+object TestData {
```

Comment:
Do we want a variety of invalid test data?  eg: key missing, key maps to wrong object type, etc.

Can also then expose:
```
val validTestData = listOf(VALID_1, VALID_2, ...)
val invalidTestData = listOf(INVALID_1, INVALID_2, ...)
```
Then in tests, just grab the lists and run them all through the test.

Reply:
Nice idea. I was also considering whether it makes sense (or even possible?) to include actual resources (for request/responses) rather than constants like this.

Any objection to me making a note to follow up on this, to avoid blocking the PR?

Reply:
Dump the json files in `src/test/assets/` and load them in the test by asset name :)